cmd/skupper/skupper_podman.go

```diff
@@ -19,6 +19,7 @@ var SkupperPodmanCommands = []string{
 
 type SkupperPodman struct {
 	cli     *clientpodman.PodmanRestClient
+	curSite *podman.Site
```

Comment:
Does this new attribute stand for "current site"? In that the case my preference would be `currentSite` because I find it more clear to me, but this is not a blocker.

Reply:
True. I will update it.